README.md

```diff
@@ -183,6 +183,41 @@ output:
 driverversion: master
 ```
 
+### redhat 7
+
+```yaml
+kernelrelease: 3.10.0-1160.66.1.el7.x86_64
+target: redhat
+output:
+  module: /tmp/falco-redhat7.ko
+driverversion: master
+builderimage: registry.redhat.io/rhel7:rhel7_driverkit
```

Comment:
Nice! A new user of `--builderimage`. The one concern I have is that this appears to be a local tag. @Issif @LucaGuerra is this an acceptable use of `--builderimage`? Not sure if we should publish the `Dockerfile` for e.g. `registry.redhat.io/rhel7:rhel7_driverkit` here as well.

Reply:
Just for info, in the examples I provided I used `registry.redhat.io/rhel7:7.9-702.1655292978` for RHEL7, `redhat/ubi8:latest` for RHEL8 and `docker.io/redhat/ubi9:latest` for RHEL9. RHEL 8 and 9 images where freely downloadable without susbscription and RHEL 7 I chose the sub one just to show as an example. I only attached my subscription info to each image and used a few stock packages (cpio, gcc, curl etc) before tagging the images like indicated in the examples further up. The dockerfile for each is pretty simple, like a few lines. If you want them uploaded somewhere I can do that and have them take RHEL user/password arguments if you think it'll help folk but I thought it was so simple I didn't include in the original PR. If you want them uploaded, just let me know where they should go in the repo tree?

Also, regarding the comment about acceptable use of `--builderimage`, was it not intended that it should be used this way e.g. pass your own image to make a build? The comment in the code said `docker image to be used to build the kernel module and eBPF probe. If not provided, the default image will be used` which I thought was the exact use case I was following for this PR?